static/js/BookPage.jsx

```diff
@@ -539,7 +533,14 @@ class SchemaNode extends Component {
         );
       } else if (this.props.schema.nodeType === "DictionaryNode") {
         return (
-          <DictionaryNode schema={this.props.schema} />
+          <DictionaryNode
+              schema={this.props.schema}
+              lexiconName={this.props.lexiconName}
```

Comment:
Where does this prop come from? I'm not seeing it passed to `SchemaNode` but I might be missing something. And shouldn't `lexiconName` be on the `schema` prop already since this is a DictionaryNode?